atom/browser/atom_browser_client.cc

```diff
@@ -184,7 +184,6 @@ void AtomBrowserClient::OverrideWebkitPrefs(
     content::RenderViewHost* host, content::WebPreferences* prefs) {
   prefs->javascript_enabled = true;
   prefs->web_security_enabled = true;
-  prefs->javascript_can_open_windows_automatically = true;
```

Comment:
@zcbenz
This pref was removed in https://chromium-review.googlesource.com/c/chromium/src/+/512347
Can we just remove it too?

Reply:
Yes we can just remove it.